### PR TITLE
Update Documentation 

### PR DESCRIPTION
**Pull request by daico007**
### PR Summary:
Update documentation removing `omnia` and `mosdef` channel. Also remove those two channels from azure yml file (except for Windows build) 

### PR Checklist
------------
 - [ ] Includes appropriate unit test(s)
 - [ ] Appropriate docstring(s) are added/updated
 - [ ] Code is (approximately) PEP8 compliant
 - [ ] Issue(s) raised/addressed?

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
 
-Install from pip:
+    $ conda config --add channels conda-forge
+    $ conda install foyer
+Install with `pip <https://pypi.org/project/pip/>`_
+---------------------------------------------------
+::
 
-.. code:: bash
+    $ pip install foyer
```
Comment (justinGilmer) [inline docs/installation.rst:19]: This should not be used anymore. Now that we are on conda-forge

**Review by justinGilmer — CHANGES_REQUESTED** [review]

File: docs/installation.rst [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+Install an editable version from source
+---------------------------------------
+::
 
-Install an editable version from source:
-
-.. code:: bash
-
-    git clone https://github.com/mosdef-hub/foyer.git
-    cd foyer
```
Comment (jennyfothergill) [inline docs/installation.rst:21]: might be nice to mention how to use ymls to install dependencies here

**Comment by rsdefever** [comment]
@daico007 I hate to throw away your work here, but I've also been updating this with my docs overhaul. I don't know if it makes sense to merge this PR before we merge the docs overhaul or not... :(

**Comment by daico007** [comment]
> @daico007 I hate to throw away your work here, but I've also been updating this with my docs overhaul. I don't know if it makes sense to merge this PR before we merge the docs overhaul or not... :(

That's completely fine! I did not have too much change on this PR anyway.

**Pull request closed without merging**